### DISCUSSION TITLE
安装报错，求大佬帮忙解决

### DISCUSSION THREAD

**Discussion by Xue-chuxin** [lsky-org/lsky-pro#677] [问答] [answered]
![QQ图片20230605151412](https://github.com/lsky-org/lsky-pro/assets/69140474/73da36e2-c5a4-47bf-b350-e41b1bdd6327)
Warning: require(/www/wwwroot/img.setonink.com/public/../vendor/autoload.php): Failed to open stream: No such file or directory in /www/wwwroot/img.setonink.com/public/index.php on line 34

Fatal error: Uncaught Error: Failed opening required '/www/wwwroot/img.setonink.com/public/../vendor/autoload.php' (include_path='.:') in /www/wwwroot/img.setonink.com/public/index.php:34 Stack trace: #0 {main} thrown in /www/wwwroot/img.setonink.com/public/index.php on line 34

**Comment by akatsukiro** [comment, marked as answer]
https://github.com/lsky-org/lsky-pro/discussions/357#discussion-3961971

**Comment by reaishijie** [comment]
下错源码了
